.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7, 3.6, 3.8]
+        python-version: [3.6, 3.8]
```

Comment:
Note to self for later - we need to add 3.9 here eventually. boost-histogram should support 3.9 in under a week, I think.

Reply:
Indeed 3.9 is freshly out. Under a week sounds great.